source/features/latest-tag-button.tsx

```diff
@@ -18,45 +30,83 @@ const getLatestTag = cache.function(async (): Promise<string | false> => {
 			}) {
 				nodes {
 					name
+					target {
+						oid
+						... on Commit {
+							committedDate
```

Comment:
Don’t compare dates, git can compare commits directly. Once you have the latest tag, you just compare “default branch” to “latest tag”, and if there’s a difference then it’s “bleeding edge”

You can get both of these information in the same GraphQL query: `{latestTag, isBleedingEdge}`

Reply:
Correction, maybe getting both piece of information in a single request isn't possible, but this is the API to see if the current branch is the latest:

```
https://api.github.com/repos/sindresorhus/refined-github/compare/master...20.2.3
```

Where `master` is the branch shown on the page and `20.2.3` is the `latestTag` info we have.

The API returns an object that has `behind_by` and `ahead_by` properties we can store and use to display exactly 

```
${branch} is ${ahead_by} commits ahead of the latest release
```

Reply:
@fregante One downside to this approach that I just discovered: The V3 API returns a lot of unnecessary information, which slows down the call. (Google tells me it's not possible to only return a few fields.. )
One of my example calls has 320 commits after the last release - retrieving that takes 2.2 seconds:
![bleeding_edge_slow_response](https://user-images.githubusercontent.com/6058517/74317838-00039000-4d74-11ea-99bb-1877cb99a334.png)
Which means that the latest-tag box only shows up way after the page has finished loading.
Is that acceptable?

One other approach I took this morning, before I saw your suggestion, was to simply get the latest commit for each branch in a single call:
```
{
  repository(name: "moto", owner: "spulec") {
    latestTag:object(expression: "1.3.14") {
      ... on Commit {
        oid
      }
    },
    defaultBranch:object(expression: "master") {
      ... on Commit {
        oid
      }
    }
  }
}
```
It doesn't give us the full picture, but it's enough for the simple 'Current branch is ahead of the latest release' message

Reply:
Just freestyling here - we can always do both?
1. Use GraphQL to check whether master is bleeding edge
2. Display the icon, so the user immediately knows
3. Make the slow V3 call to get the specifics
4. Update the icon label accordingly

Reply:
I didn’t think it would be so slow. 
I think your single-call solution is best, at least until there’s an equivalent GraphQL `compare` API. I can’t believe some things are so difficult with GraphQL

Reply:
Done @fregante. I also changed the logic so that only the default branch can be bleeding edge

> Just a note: by "bleeding edge" I meant "on master but not tagged", so a tag cannot be marked as "bleeding edge"